.github/workflows/pr_badges.yaml

```diff
@@ -0,0 +1,65 @@
+name: PR badges
+
+on:
+  pull_request_target:
+    branches:
+      - 'main'
+      - '23.lts.1\+'
+      - '22.lts.1\+'
+      - '21.lts.1\+'
+      - '20.lts.1\+'
+      - '19.lts.1\+'
+      - 'rc_11'
+      - 'COBALT_9'
+
+concurrency:
+  group: '${{ github.workflow }}-${{ inputs.platform }} @ ${{ github.event.pull_request.head.label || github.head_ref || github.sha }}'
+  cancel-in-progress: true
+
+permissions:
+  pull-requests: write
+
+jobs:
+  comment:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/github-script@v6
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          script: |
+            // Get the existing comments.
+            const {data: comments} = await github.rest.issues.listComments({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              issue_number: context.payload.number,
+            })
+
+            // Find any comment already made by the bot.
+            const botComment = comments.find(comment => {
+              return comment.user.type === 'Bot' && comment.body.includes('Build Status')
+            })
+            const commentBody = `
+            ## Build Status
+            | Workflow  | Status                                                                                                                                                                                                                                               |
+            | --------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+            | lint      | [![lint](${{github.server_url}}/${{github.repository}}/actions/workflows/lint.yaml/badge.svg?branch=${{ github.head_ref }})](${{github.server_url}}/${{github.repository}}/actions/workflows/lint.yaml?query=branch%3A${{ github.head_ref }})                |
```

Comment:
Consider adding each check to this body string in a loop over the checks (i.e. platforms + lint check) if possible

Reply:
Done